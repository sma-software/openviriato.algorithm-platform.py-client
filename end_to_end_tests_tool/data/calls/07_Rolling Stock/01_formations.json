{
    "restResource": "/vehicles/formations/{formationId}",
    "httpMethod": "GET",
    "urlParameters": [ "=multiTrain:trainPathNodes[0].formationId=" ],
    "requestBody": null,
    "CSharpMethodName": "GetFormation(formationID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory" : "Get Formation" 
}
