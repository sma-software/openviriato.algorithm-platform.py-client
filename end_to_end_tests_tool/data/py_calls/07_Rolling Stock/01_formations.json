{
  "PythonParameterMapping": {
    "formation_id": "$.urlParameters.[0]"
  }
}