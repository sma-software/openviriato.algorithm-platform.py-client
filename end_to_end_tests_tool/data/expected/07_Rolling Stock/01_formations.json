{
  "id": 4108,
  "vehicleTypeIds": [
    4107
  ],
  "placesFirstClass": 125,
  "placesSecondClass": 326,
  "debugString": "train configuration:  seats 125 326"
}